### PR TITLE
Friends page

### PR DESCRIPTION
**Pull request by bk1405bgl**
Friends page is created and the form works well.

please check and approve for QA

### CODE REVIEW

File: pages/friends.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import Form from "./components/Form";
+import React from "react";
+import { useState } from "react";
+import { uid } from "uid";
+
+export default function Friends() {
+  const [friends, setFriends] = useState([]);
+
+  function handleAdd(newFriend) {
+    console.log(friends);
```
Comment (Kheirah) [inline pages/friends.js:12]: Remove logs.

**Review by Kheirah — CHANGES_REQUESTED** [review]
Good start!

Your form is working well. But you need to change your file structure. Currently, you have `components` within the `pages` folder. Drag `components` up to place it at the root level of your project.

File: pages/friends.js (hunk 1 above)
Comment (bk1405bgl) [reply pages/friends.js:12]: ok, removed.

File: pages/components/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    event.target.elements.fName.focus();
+  }
+
+  return (
+    <>
+      <h2>Freund hinzufügen</h2>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <label htmlFor="fName">Name: </label>
+          <input id="fName" type="text" name="fName" required />
```
[the remark this answers is not in the document]
Comment (bk1405bgl) [reply pages/components/Form/index.js:19]: It should describe friendsName, i changed that.
the components folder is now one level up, and the app still works.
and i removed the console.log.

File: pages/friends.js (hunk 1 above)
Comment (bk1405bgl) [reply pages/friends.js:12]: i removed already.
should i make a new code-review?

**Review by Kheirah — APPROVED** [review]
Perfect!

**Pull request merged**